render.yaml

```diff
@@ -0,0 +1,23 @@
+databases:
+  - name: postgres
+    ipAllowList: []
+    plan: free # optional (defaults to starter)
+
+services:
+  - type: web
+    name: rails
+    env: ruby
+    buildCommand: "./bin/render-build.sh"
+    startCommand: "bundle exec rails s"
+    envVars:
+      - key: DATABASE_URL
+        fromDatabase:
+          name: postgres
+          property: connectionString
+      # - key: RAILS_MASTER_KEY
```

Comment:
OK! I think I remember now what the master key stuff is about. We don't use `config/credentials.yml.enc` so we should not need this.

Reply:
That said, every neighborhood needs to generate a random `LOCKBOX_MASTER_KEY`